frontend/src/metabase/admin/datamodel/components/filters/pickers/DatePicker/DatePickerHeader.styled.tsx

```diff
@@ -40,7 +44,7 @@ export const BackButton = styled(TabButton)`
   border: none;
   border-radius: 0;
   margin-left: ${space(1)};
-  color: var(--mb-color-text-medium);
+  color: var(--mb-color-text-secondary);
```

Comment:
Fix the back button color
![Screenshot 2024-08-01 at 2 52 23 PM](https://github.com/user-attachments/assets/8ac32647-d552-4c8b-bcfb-be4c4bbe0efd)